app/controllers/concerns/authenticatable.rb

```diff
@@ -59,9 +64,12 @@ def prepare_authorization_via_shared_access_token
   def doorkeeper_sign_in
     token = token_authentication.oauth_token
     return unless token&.accessible?
-    return head(:forbidden) unless token.acceptable?(:api)
-
-    sign_in token.person, store: false
+    acceptable = token.acceptable?(:people) ||
+      token.acceptable?(:groups) ||
+      token.acceptable?(:events) ||
+      token.acceptable?(:api)
+    return head(:forbidden) unless acceptable
+    sign_in token, store: false
```

Comment:
To be verified: Not signing in as `token.person` anymore (but defining the abilities via `DoorkeeperTokenAbility`) might limit the `:api` scope in some places. Currently, reading people, group, event and invoice data is supported. This covers everything currently mentioned in https://github.com/hitobito/hitobito/blob/master/doc/development/05_rest_api.md. Is there anything else?

Reply:
I went through all the serializers in app/serializers and looked for ones that are used in controllers. It's the following list currently:
`AppStatusSerializer` (only used for application health checks), `EventParticipationSerializer`, `EventSerializer`, `GroupListSerializer`, `GroupSerializer`, `InvoiceSerializer`, `ListSerializer` (used together with other serializers for rendering lists instead of single models), `MailingListSerializer`, `PeopleSerializer`, `PersonSerializer`, `UserSerializer` (used for the deprecated User API Token endpoints).
Out of these, I think you are only missing the mailing list endpoints, which haven't been documented yet in 05_rest_api.md.

Reply:
Added mailing list scope in b06e5b8